From: Bruce Allan <bruce.w.allan@intel.com>
Date: Mon, 10 May 2010 14:59:10 +0000
Subject: e1000e: use static params to save stack space (part 2)
References: bnc#699092, fate#311825
Patch-mainline: v2.6.35
Git-commit: 4fe4491fc5578019174d0f02d1ae740fce78deac

A couple stack cleanups missed in an earlier patch from Jesse.

Signed-off-by: Bruce Allan <bruce.w.allan@intel.com>
Cc: Jesse Brandeburg <jesse.brandeburg@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Cho, Yu-Chen <acho@suse.com>
---
 drivers/net/e1000e/param.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/net/e1000e/param.c
+++ b/drivers/net/e1000e/param.c
@@ -290,7 +290,7 @@ void __devinit e1000e_check_options(stru
 		}
 	}
 	{ /* Receive Interrupt Delay */
-		struct e1000_option opt = {
+		static struct e1000_option opt = {
 			.type = range_option,
 			.name = "Receive Interrupt Delay",
 			.err  = "using default of "
@@ -390,7 +390,7 @@ void __devinit e1000e_check_options(stru
 		}
 	}
 	{ /* Interrupt Mode */
-		struct e1000_option opt = {
+		static struct e1000_option opt = {
 			.type = range_option,
 			.name = "Interrupt Mode",
 			.err  = "defaulting to 2 (MSI-X)",
